lib/src/models/chat_feedback_info.dart

```diff
@@ -0,0 +1,15 @@
+class ChatFeedbackInfo {
+  final String authorId;
+  final int rating;
+  final String? comment;
+
+  ChatFeedbackInfo({required this.authorId, required this.rating, this.comment});
+
+  factory ChatFeedbackInfo.fromJson(Map<String, dynamic> json) {
```

Comment:
In other places, we use a named constructor instead of a factory. Why use a factory here?

```dart
  ChatFeedbackInfo.fromJson(Map<String, dynamic> json)
      : authorId = json['authorId'],
        rating = json['rating'],
        comment = json['comment'];
```

https://medium.com/nerd-for-tech/named-constructor-vs-factory-constructor-in-dart-ba28250b2747

Reply:
Make sense. Thank you